### ISSUE TITLE
image must not be empty

### ISSUE THREAD

**Issue by z3ntu** [terraform-providers/terraform-provider-digitalocean#144]
### Terraform Version
```
Terraform v0.11.8
+ provider.digitalocean v1.0.0
```

I have an existing droplet that I imported into terraform a while ago but now (after upgrading to 1.0.0 probably) I can't get around the `Error: digitalocean_droplet.vps: image must not be empty` error I get with basically every command. If I change the `image` value to anything other than an empty string, I get `      image:                "" => "test" (forces new resource)` when I call `terraform plan` but I don't want to recreate my droplet so I would like to keep it at an empty string.

EDIT: The API returns the following block for my image:
```
            "image": {
                "id": 15542960,
                "name": "arch-configured",
                "distribution": "Debian",
                "slug": null,
                "public": false,
                "regions": [
                    "ams3"
                ],
                "created_at": "2016-01-29T14:03:45Z",
                "min_disk_size": 20,
                "type": "snapshot",
                "size_gigabytes": 1.7
            },
```

**Comment by z3ntu** [comment]
Thanks for the fix! I've built the binary from git and replaced it, ran `env TF_SKIP_PROVIDER_VERIFY=1 terraform refresh` and now the image string is some number which I can use in the config.

Is there a better way to get the updated provider into my terraform folder (so I don't have to use this environment variable)?

**Comment by andrewsomething** [comment]
No problem! Thanks for the report. We'll be cutting a new 1.0.1 release with this fix, hopefully before the end of the day.

**Comment by andrewsomething** [comment]
This fix was released as version 1.0.1 earlier today. You can upgrade with:

```
terraform init -upgrade
```

Thanks again for the report.
